frontend/legacyPartCollectionRoutes.js

```diff
@@ -66,11 +68,13 @@ const legacyTagsWithNoItemType = [
    'PC_Cards',
    'PCIe',
    'Propellers',
+   'Prying_and_Opening',
    'Pulleys',
    'Pumps',
    'Retail_Display',
    'RJ-11_Boards',
    'SD_Card_Slots',
+   'Sensors',
```

Comment:
Looks like this is actually valid, and we don't want a comprehensive redirect: https://www.ifixit.com/Parts/Samsung_Galaxy_S9/Sensors

```suggestion
```